.github/workflows/docs.yml

```diff
@@ -0,0 +1,25 @@
+name: Docs
+
+on: [push, pull_request]
+
+jobs:
+    docs:
+        name: Auto generate docs
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v4
+
+            - uses: kdheepak/panvimdoc@main
+              with:
+                vimdoc: harpoon
+                pandoc: README.md
+                description: Best plugin ever created
```

Comment:
The description here should probably say, "Getting you where you want with the fewest keystrokes"